.github/workflows/dockerfile-llb-frontend.yaml

```diff
@@ -0,0 +1,36 @@
+name: tools/dockerfile-llb-frontend
+
+on:
+  push:
+    branches: [stable]
+    paths:
+      - 'tools/dockerfile-llb-frontend/**'
+      - '.github/workflows/dockerfile-llb-frontend.yaml'
+
+jobs:
+  plugin-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Registry
+        uses: docker/login-action@v2
+        with:
```

Comment:
Only login on event push and not when PRs are submitted:
```diff
+ if: ${{ github.event_name == 'push' }}
```

Reply:
Done